lib/driver.rb

```diff
@@ -0,0 +1,83 @@
+require_relative 'csv_record'
+
+module RideShare
+  class Driver < CsvRecord
+
+  attr_reader :name, :vin, :trips, :status
+  
+  def initialize(id:, name:, vin:, status: :AVAILABLE, trips: nil)
+    super(id)
+
+    @name = name
+    @vin = vin
+    @status = status.to_sym
+    @trips = trips || []
+  
+    vin_pattern = /^[A-Z0-9]{17}$/i
+    if (vin_pattern =~ @vin) == nil
+      raise ArgumentError, "Invalid VIN"
+    end
+  end
+
+  def add_trip(trip)
+    @trips << trip
+  end
+
+  def change_status
+    @status = :UNAVAILABLE
+  end
+
+  # Wave 2 get average rating for a driver
+  def average_rating
+    nil_cases = 0
+
+    if @trips.empty? == true
```

Comment:
You don't need to compare boolean values to `true`:

```suggestion
    if @trips.empty?
```